client/state/products-list/selectors/compute-full-and-monthly-prices-for-plan.ts

```diff
@@ -36,7 +53,7 @@ export const computeFullAndMonthlyPricesForPlan = ( state, siteId, planObject )
 		? getIntroOfferPrice( state, planObject.getProductId(), siteId )
 		: null;
 	const saleCouponCost = isJetpackSearchProduct
-		? Math.floor( planOrProductPrice * ( 1 - saleCouponDiscount ) * 100 ) / 100
```

Comment:
Since `planOrProductPrice` can be null, I have no idea how this didn't explode before.